packages/angular/cli/commands/e2e-impl.ts

```diff
@@ -14,8 +14,14 @@ import { Schema as E2eCommandSchema } from './e2e';
 export class E2eCommand extends ArchitectCommand<E2eCommandSchema> {
   public readonly target = 'e2e';
   public readonly multiTarget = true;
+  public readonly missingTargetError = `
+Cannot find "e2e" target for the specified project.
 
-  public async run(options: E2eCommandSchema & Arguments) {
-    return this.runArchitectTarget(options);
+You should add a package that implements e2e capabilities.`;
```

Comment:
You should add a package that implements end-to-end testing capabilities.

Reply:
done